src/main/java/com/graphaware/module/es/ElasticSearchConfiguration.java

```diff
@@ -36,6 +36,9 @@
     private static final InclusionPolicies DEFAULT_INCLUSION_POLICIES = InclusionPoliciesFactory.allBusiness().with(IncludeNoRelationships.getInstance());
     private static final Mapping DEFAULT_MAPPING = DefaultMapping.newInstance();
     private static final boolean DEFAULT_ASYNC_INDEXATION = false;
+    public static final int DEFAULT_READ_TIMEOUT = 20000; //20s
+    public static final int DEFAULT_CONNECTION_TIMEOUT = 10000; //10s
```

Comment:
Same here

Reply:
Updated in accordance to 

`        protected Integer connTimeout = 3000;
        protected Integer readTimeout = 3000;`

from Jest's ClientConfig.java